src/collect/cli.rs

```diff
@@ -58,6 +61,9 @@ Example: --probe tp:skb:kfree_skb --probe kprobe:consume_skb"
         default_value = "false"
     )]
     pub(super) print: bool,
+    #[arg(long, help = "Format used when printing an event.")]
+    #[clap(value_enum, default_value_t=DisplayFormat::MultiLine)]
+    pub(super) format: DisplayFormat,
```

Comment:
tiny nit: I find more readable if help is just the doc-comment. feel free to ignore as the rest of the fields don't have.